### DISCUSSION TITLE
GhostLight: A level editor for Excalibur that runs in your browser

### DISCUSSION THREAD

**Discussion by simon-jaeger** [excaliburjs/Excalibur#2159] [Show and tell]
Hey everyone, nice to meet ya! I'm Simon, and over the last few weeks, I've started working on a new level editor called *GhostLight*. It runs directly in your browser and can, of course, be used in combination with Excalibur to make games.

To use it, visit https://ghostlight.onrender.com with an up to date version of Google Chrome.

<img width="1086" alt="GhostLight" src="https://user-images.githubusercontent.com/17790318/146564148-a8524579-fa4c-4a07-bdcf-06f535c3745c.png">

I've also built a little demo of how you may use Excalibur to bring a level designed in GhostLight to life:
[Play it online](https://ghostlight-excalibur-demo.onrender.com/)
[Get the source code](https://github.com/simon-jaeger/ghostlight-excalibur-demo)

![ghostlight-demo-5](https://user-images.githubusercontent.com/17790318/146564217-c5238d80-aaec-4b00-b4e7-eb72e1cc3df6.gif)

I hope to build a tool which will make game development with Excalibur even more awesome! Let me know what you think. GhostLight is still in an early stage and I need your feedback – thanks!

**Comment by eonarheim** [comment]
Bravo @simon-jaeger! This is absolutely fantastic!

**Comment by simon-jaeger** [comment]
Thank you @eonarheim and everyone else for the positive reactions. I'll keep working on the editor. For those interested, here's a little video tutorial:

[![GhostLight](https://img.youtube.com/vi/WMHZLPA_Cho/0.jpg)](https://www.youtube.com/watch?v=WMHZLPA_Cho) <br>
[→ YouTube](https://www.youtube.com/watch?v=WMHZLPA_Cho)

**Comment by kgish** [comment]
Good start! Can't wait for more advanced features. Don't make it too complicated though.
